### DISCUSSION TITLE
Would love to see a count of the total classes being used

### DISCUSSION THREAD

**Discussion by alpacachen** [unocss/unocss#3996] [Q&A] [answered]
how to do this

**Comment by userquin** [comment, marked as answer]
You can enable `inspector: true,` navigate the app and then open a new tab with `/__unocss` url: for example `http://localhost:5173/__unocss`

https://unocss.dev/tools/inspector#inspector

**Reply by alpacachen** [reply]
thx bro

**Reply by alpacachen** [reply]
hi,another question herer
https://github.com/unocss/unocss/issues/3669
so i disable the html plugin
the page show like this, no file infos
<img width="1884" alt="image" src="https://github.com/user-attachments/assets/17b8d63d-53d8-41df-b095-47300791c232">
how can i solve this?

**Reply by userquin** [reply]
Are you using only js or ts files?

**Reply by alpacachen** [reply]
> Are you using only js or ts files?

i found problem， vite html plugin
